sdk-setup/src/mb2

```diff
@@ -1856,104 +1856,107 @@ run_build_requires__process_args() {
     fi
 }
 
+sign_rpms__prepare() {
+    SIGN_SDKSIGN=/usr/libexec/sdk-setup/sdk-rpmsign
```

Comment:
This would be better set at file level here https://github.com/sailfishos/sdk-setup/blob/1.3.88/sdk-setup/src/mb2#L3032